src/modules/schema.graphql

```diff
@@ -1,3 +1,11 @@
 type Query {
     getMyProfile: User!
+    getAllTrainees: [User]
+    getTrainee(id: ID!): User!
+}
+
+type Mutation {
+    createTrainee(user: CreateUserInput): User!
+    updateTrainee(id: ID!, role: String, name: String, email:String): User!
```

Comment:
change this **updateTrainee(id: ID!, role: String, name: String, email:String): User!**
to 
**updateTrainee(updateUser: UpdateUserInput): User!**
and create type of UpdateUserInput and also need to change in the resolver for this as well